### PR TITLE
Add archive.org and img data: CSP

### PR DESCRIPTION
**Pull request by Gregoor**

### CODE REVIEW

**Review by escattone — APPROVED** [review]
Thanks @Gregoor!

File: libs/constants/index.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ const CSP_DIRECTIVES = {
   ],
   "img-src": [
     "'self'",
+    "data:",
```
Comment (peterbe) [inline libs/constants/index.js:102]: Where is that even being used? Can we consider fixing all occurrences of that in the `content`  (and `translated-content`)?
I can't imagine the security risk of allowing `data:...` inside `img` tags, but it still makes me nervous. 

Also,:
```
▶ rg -t html 'data:' | rg '<img'

```
empty! True for `translated-content` too.

Comment (peterbe) [reply libs/constants/index.js:102]: I'm confused! https://developer.allizom.org/en-US/docs/Learn/HTML/Cheatsheet supposedly triggers the CSP violation for `img-src` `data:`. 
https://sentry.prod.mozaws.net/operations/mdn-stage/issues/11335946/?query=is%3Aunresolved
But it's [nowhere to be found in the source](https://github.com/mdn/content/blob/main/files/en-us/learn/html/cheatsheet/index.html).

Comment (escattone) [reply libs/constants/index.js:102]: > I'm confused! https://developer.allizom.org/en-US/docs/Learn/HTML/Cheatsheet supposedly triggers the CSP violation for `img-src` `data:`.
> https://sentry.prod.mozaws.net/operations/mdn-stage/issues/11335946/?query=is%3Aunresolved
> But it's [nowhere to be found in the source](https://github.com/mdn/content/blob/main/files/en-us/learn/html/cheatsheet/index.html).

Good point about the `data:` URL's @peterbe. I don't understand this as well. I don't see that CSP violation when I load that page either.

**Pull request merged**